src/tests/ftest/osa/osa_offline_reintegration.py

```diff
@@ -24,111 +25,195 @@ def setUp(self):
+                # Check pool version incremented after pool exclude
+                # pver_exclude should be greater than
+                # pver_begin + 3 (2 targets + exclude)
+                self.assertTrue(pver_exclude > (pver_begin + 3),
+                                "Pool Version Error: After exclude")
+
+            # Reintegrate the ranks which was excluded
+            for val, _ in enumerate(rank):
+                time.sleep(5)
+                if (val == 2 and "RP_2G" in oclass):
+                    output = self.dmg_command.pool_reintegrate(self.pool.uuid,
+                                                               rank[val],
+                                                               "0,2")
+                else:
+                    output = self.dmg_command.pool_reintegrate(self.pool.uuid,
+                                                               rank[val])
                 self.log.info(output)
-                output = self.dmg_command.system_start(ranks=rank)
-
-            self.log.info(output)
-            self.is_rebuild_done(3)
-            self.assert_on_rebuild_failure()
-
-            pver_exclude = self.get_pool_version()
-            self.log.info("Pool Version after exclude %s", pver_exclude)
-            # Check pool version incremented after pool exclude
-            # pver_exclude should be greater than
-            # pver_begin + 8 targets.
-            self.assertTrue(pver_exclude > (pver_begin + 8),
-                            "Pool Version Error:  After exclude")
-            output = self.dmg_command.pool_reintegrate(self.pool.uuid,
-                                                       rank)
-            self.log.info(output)
-            self.is_rebuild_done(3)
-            self.assert_on_rebuild_failure()
-
-            pver_reint = self.get_pool_version()
-            self.log.info("Pool Version after reintegrate %d", pver_reint)
-            # Check pool version incremented after pool reintegrate
-            self.assertTrue(pver_reint > (pver_exclude + 1),
-                            "Pool Version Error:  After reintegrate")
+                self.is_rebuild_done(3)
+                self.assert_on_rebuild_failure()
 
-        for val in range(0, num_pool):
-            display_string = "Pool{} space at the End".format(val)
-            self.pool = pool[val]
-            self.pool.display_pool_daos_space(display_string)
+                pver_reint = self.get_pool_version()
+                self.log.info("Pool Version after reintegrate %d", pver_reint)
+                # Check pool version incremented after pool reintegrate
+                self.assertTrue(pver_reint > (pver_exclude + 1),
+                                "Pool Version Error:  After reintegrate")
 
-        if data:
-            self.run_ior_thread("Read", self.ior_dfs_oclass[0],
-                                self.ior_apis[0], self.ior_test_sequence[0])
```

Comment:
This is for verifying the data, why remove that?
btw: can this pass with master?

Reply:
No. I didn't remove it... We are doing more data check now. ior and mdtest...

![image](https://user-images.githubusercontent.com/48604608/111000227-a6431000-834f-11eb-9087-0f2b54696fa5.png)